src/util.test.ts

```diff
@@ -38,7 +38,12 @@ describe("Util", (): void => {
       expect(core.info).nthCalledWith<[string]>(1, command);
       expect(child_process.exec).lastCalledWith(
         command,
-        { shell: "/usr/bin/bash" },
+        {
+          shell:
+            process.platform === "win32"
+              ? "C:\\Program Files\\Git\\bin\\bash.exe"
+              : "/usr/bin/bash",
+        },
```

Comment:
Nit (multiple instances): it's probably worth having this defined in its own `execOptions` variable for readability. The less the brain has to absorb per line, the easier (within reason).

Reply:
I made a `shellPath` variable, which I had done in some of the code, to simplify it.  I hope that accomplishes what you are hoping for.  Let me know. It seemed like `execOptions` would end up being pretty wordy if I didn't introduce an extra variable in some parts of the code.